### ISSUE TITLE
Improve layout of admin forms

### ISSUE THREAD

**Issue by Firgrep** [systemphil/sphil#108]
- Admin forms or even the entire admin section could use a face-lift.

**Comment by nafisreza** [comment]
Hello @Firgrep ,
I have set up the repo locally to work on this. But haven't found any admin form. Could you tell me more about the section that needs to change?

**Comment by Firgrep** [comment]
That's because it's in the "v1" working branch `firgrep/nextra-4-alpha` https://github.com/systemphil/sphil/pull/98 

The routes are in `app/(admin)/admin` 

You'll find the forms themselves in `features/courses` 

I don't think you'll be able to run the app without the db and auth keys. I have updated the readme on the branch on how to set up a local db instance; from there you can just use a random generated string for the auth secret and make some github provider credentials from the github developer settings. 

I'll try to make this easier and more convenient for quick local development setup, but as it is, there's a lot of work on that branch.

**Comment by nafisreza** [comment]
Hey, it's my pleasure to contribute. 😃
Btw, I went to your `firgrep/nextra-4-alpha` branch and tried to run the dev server. But looks like the `page` and `app` directory are colliding. So I deleted the `index.mdx` in old `pages` directory and now the frontend is running fine. 
But I guess will still need some help setting up the backend as it is throwing error in the auth pages. Let's connect on some platform to have some chitchats maybe?

**Comment by Firgrep** [comment]
Yes, the old `src` folder will be deleted; it's just there so that the current main branch can merge more easily into it. 

Sure, send us a note on our email service@systemphil.com and we'll send you a link to our discord. :)
